zerver/lib/export.py

```diff
@@ -1739,7 +1737,7 @@ def get_analytics_config() -> Config:
     # analytics.json file in a full-realm export.
 
     analytics_config = Config(
-        table='zerver_analytics',
+        table='zerver_realm',
         is_seeded=True,
     )
```

Comment:
While this is an improvement, I think the actual bug here is that `analytics.json` shouldn't have a copy of `zerver_realm` at all.  

The fix might be this:
```
+    # The seeding logic results in a duplicate zerver_realm object
+    # being included in the analytics data.  We don't want it, as that
+    # data is already in `realm.json`, so we just delete it here
+    # before writing to disk.
+    del response['zerver_realm']

```